src/nvim/if_cscope.c

```diff
@@ -127,7 +127,7 @@ char_u *get_cscope_name(expand_T *xp, int idx)
       if (csinfo[i].fname == NULL)
         continue;
       if (current_idx++ == idx) {
-        vim_snprintf(connection, sizeof(connection), "%d", i);
+        vim_snprintf(connection, sizeof(connection), "%lu", i);
```

Comment:
Yes, I used "%zu" where the regular `printf` functions were used, but on line 3042 in `message.c` there is a comment about what `snprintf` supports, and it has no mention of "%zu". I thought that "%lu" would be the next best thing. If it in fact does support "%zu" then I'll change it, I did not delve in to the source to check, but I could do that to verify.